src/photon/photon_scheduler.c

```diff
@@ -187,7 +190,9 @@ void free_local_scheduler(local_scheduler_state *state) {
  */
 void start_worker(local_scheduler_state *state, actor_id actor_id) {
   /* We can't start a worker if we don't have the path to the worker script. */
-  CHECK(state->config.start_worker_command != NULL);
+  if (state->config.start_worker_command == NULL) {
```

Comment:
maybe give another warning here (you gave one at the beginning, but it might be lost in the logs now)